src/routes/users.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import UsersController from '../controllers/usersController';
+import authMiddleware from '../middlewares/auth';
+import { profileSchema } from '../validation/schemas';
+import validate from '../validation/validator';
+
+const router = express.Router();
+
+router.get('/me', authMiddleware, UsersController.getProfile);
+router.patch('/me', authMiddleware, validate(profileSchema, 'body'), UsersController.updateProfile);
```

Comment:
Instead of using '/me' ,  i think  '/profile' is a more descriptive name for this  route